docs/docs/kafka.md

```diff
@@ -4,10 +4,12 @@ Any options given to Maxwell that are prefixed with `kafka.` will be passed dire
 (with `kafka.` stripped off).  We use the "new producer" configuration, as described here:
 [http://kafka.apache.org/documentation.html#newproducerconfigs](http://kafka.apache.org/documentation.html#newproducerconfigs)
 
-Maxwell sets the following Kafka options by default, but you can override them in `config.properties`.
+Maxwell sets the following Kafka options by default via the properties file. You can remove or update them in `config.properties`.
 
 - kafka.acks = 1
 - kafka.compression.type = gzip
+- kafka.metadata.fetch.timeout.ms=50000
+- kafka.retries=0
```

Comment:
should change the verbage a bit; you can just say "these are recommended properties" or something.  I'll also put some examples of more strongly consistent setups and such after I merge.